dev/deps.py

```diff
@@ -292,8 +292,8 @@ def _extract_package(deps_dir, pkg_path, pkg_dir):
                     dst_dir = os.path.dirname(dst_path)
                     if not os.path.exists(dst_dir):
                         os.makedirs(dst_dir)
-                    with open(dst_path, 'wb') as f:
```

Comment:
`deps.py` is actually shared across a number of different Python projects I write, some of which still support Python 2, so I'd prefer to leave it alone so I can more easily migrate fixes in the future.